### ISSUE TITLE
shut console with logging or error 

### ISSUE THREAD

**Issue by shlomoweb1** [oracle/oci-typescript-sdk#266]
I am getting this message.
```
"The OciEnabledServiceSet is empty, all OCI services are enabled"
ObjectStorageClient endpoint set to https://objectstorage.il-jerusalem-1.oraclecloud.com
Calling operation ObjectStorageClient#createPreauthenticatedRequest.
Retry policy to use: MaximumNumberAttempts=7, MaxSleepBetween=30, ExponentialBackoffBase=2
Total Latency for this API call is: 46 ms
Total Latency for this API call is: 45 ms
Total Latency for this API call is: 38 ms
```

It is very annoying, and I do not want to have such messages in production; all the functions work, and I can't figure out why I keep seeing these messages

**Comment by jyotisaini** [comment]
Hi @shlomoweb1 Have you by any changes enabled logging info ?

**Comment by shlomoweb1** [comment]
This is my `oci_config` file.
```
[DEFAULT]
user=ocid1.user.oc1..aaaaaaaa3ddudh5i6akkpaxedhgdueunrbxlbfhbc3sfr5ir7nj3ikmiit5a
fingerprint=d5:a1:26:35:53:e4:9a:63:de:d0:a9:60:ba:41:74:48
tenancy=...some-tenancy
region=il-jerusalem-1
key_file=[location of pem file]
```

This is my wrapper to generate access keys to the bucket (object storage); I have nowhere else set for OCI, and the code work

```
import * as common from "oci-common";
import * as objectstorage from "oci-objectstorage";
import path from 'path';
import moment from 'moment';

// https://docs.oracle.com/en-us/iaas/api/#/en/objectstorage/20160918/PreauthenticatedRequest/CreatePreauthenticatedRequest


// Load OCI configuration from file
const configurationFilePath = path.join(__dirname, "oci_config"); // Update the path to your configuration file
const configProfile = "DEFAULT";
const provider: common.ConfigFileAuthenticationDetailsProvider = new common.ConfigFileAuthenticationDetailsProvider(
    configurationFilePath,
    configProfile
);

export const namespaceName = 'someNameSpace';
export const defaultBucket = "bucketName";
export const defaultRegion = 'il-jerusalem-1';

const client = new objectstorage.ObjectStorageClient({
    authenticationDetailsProvider: provider
}, {});

// Function to generate a presigned URL
// https://docs.oracle.com/en-us/iaas/tools/typescript-sdk-examples/2.74.0/objectstorage/CreatePreauthenticatedRequest.ts.html
interface generatePresignedURLProps {
    bucketName?: string;
    objectName: string;
    expirationInSeconds: number;
}
export const generatePresignedURL = async ({ bucketName, objectName, expirationInSeconds }: generatePresignedURLProps) => {
    try {

        // Create a service client
        // const client = new objectstorage.ObjectStorageClient({
        //     authenticationDetailsProvider: provider
        // });

        const bucketListingAction = objectstorage.models.PreauthenticatedRequest.BucketListingAction.Deny;
        const accessType = objectstorage.models.CreatePreauthenticatedRequestDetails.AccessType.AnyObjectReadWrite;
        const timeExpires = moment().add(expirationInSeconds, 'seconds').toDate();
        const random4DigitNumber = Math.floor(Math.random() * (9999 - 99 + 1)) + 99;


        // https://docs.oracle.com/en-us/iaas/api/#/en/objectstorage/20160918/PreauthenticatedRequest/CreatePreauthenticatedRequest
        const createPreauthenticatedRequestRequest: objectstorage.requests.CreatePreauthenticatedRequestRequest = {
            namespaceName,
            bucketName: bucketName || defaultBucket,
            createPreauthenticatedRequestDetails: {
                accessType,
                name: `par-object-${moment().format('yyyymmDD')}-${random4DigitNumber.toString().padStart(4, '0')}`,
                bucketListingAction,
                objectName,
                timeExpires
            },
            opcClientRequestId: (+new Date()).toString()
        };

        // Send request to the Client.
        const createPreauthenticatedRequestResponse = await client.createPreauthenticatedRequest(
            createPreauthenticatedRequestRequest
        );
        return createPreauthenticatedRequestResponse;
    } catch (error) {
        console.log("createPreauthenticatedRequest Failed with error  " + error);
        throw new Error('שגיאה בהרשאות יצירת אישור');
    }
};
type generateViewURLprops = {
    nameSpace: string;
    bucketName: string;
    objectName: string;
    name: string;
    expirationInSeconds: number;
}
export const generateViewURL = async (props: generateViewURLprops) => {
    const bucketListingAction = objectstorage.models.PreauthenticatedRequest.BucketListingAction.Deny;
    const accessType = objectstorage.models.CreatePreauthenticatedRequestDetails.AccessType.ObjectRead;
    const timeExpires = moment().add(props.expirationInSeconds, 'seconds').toDate();

    // https://docs.oracle.com/en-us/iaas/api/#/en/objectstorage/20160918/PreauthenticatedRequest/CreatePreauthenticatedRequest
    const createPreauthenticatedRequestRequest: objectstorage.requests.CreatePreauthenticatedRequestRequest = {
        namespaceName: props.nameSpace,
        bucketName: props.bucketName,
        createPreauthenticatedRequestDetails: {
            accessType,
            name: props.name,
            bucketListingAction,
            objectName: props.objectName,
            timeExpires
        },
        opcClientRequestId: (+new Date()).toString()
    };

    try {
        // Create a service client
        const client = new objectstorage.ObjectStorageClient({
            authenticationDetailsProvider: provider
        });
        // Send request to the Client.
        const createPreauthenticatedRequestResponse = await client.createPreauthenticatedRequest(
            createPreauthenticatedRequestRequest
        );
        return createPreauthenticatedRequestResponse;
    } catch (error) {
        console.log("generateViewURL Failed with error  " + error);
        throw new Error('שגיאה בהרשאות יצירת אישור');
    }
}

// Usage
//const presignedURL = generatePresignedURL({ bucketName: "YOUR_BUCKET_NAME", objectName: "ANYTHING", expirationInSeconds: 300, fileName: "test-file.txt" }); // 5 minutes

```

**Comment by jyotisaini** [comment]
Thanks for sharing @shlomoweb1 . We have figured out the issue. We are currently working on fixing it. I'll update here once its fixed.

**Comment by shlomoweb1** [comment]
@jyotisaini , is there any update on this issue? Thanks.

**Comment by NiviPari** [comment]
Fix available from 2.84.2 onwards

**Comment by greg-md** [comment]
@NiviPari I have upgraded the sdk to latest version and still can not get rid of this annoing log:
```sh
[2024-08-19T14:26:24.176Z] [LOG] The OciEnabledServiceSet is empty, all OCI services are enabled
```

What should I do to disable it?
